release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2151,6 +2216,9 @@ spec:
 +
 For more information, see (link:https://bugzilla.redhat.com/show_bug.cgi?id=1878685[*BZ#1878685*]).
```

Comment:
Suggested edit for consistency:

For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1878685[*BZ#1878685*].